lib/rules/key-spacing.js

```diff
@@ -197,13 +207,17 @@ module.exports = function(context) {
             }
 
             width = widths[i];
-
-            if (align === "value") {
+            if (isSingleLine(context.getSource(node))) { // Only one line
```

Comment:
This was the first method I tried (actually it has to be `node.loc.end.line - node.loc.start.line === 1`),  but I seemed to have some trouble with it not always working. I'll try it again, maybe it was because of other problems in my code at that point.